src/app/discord/page.tsx

```diff
@@ -7,7 +7,7 @@ export default function DiscordInvite() {
           <div className="text-white text-center px-4 rounded-xl">
```

Comment:
Nie jestem fanem wyśrodkowania takiego tekstu, bo ciężko się to czyta. Może podział strony na dwie kolumny - z lewej treść z prawej grafika?

To w sumie uwaga do wszystkich podstronek.